ovos_PHAL_plugin_wallpaper_manager/__init__.py

```diff
@@ -20,15 +19,20 @@ def __init__(self, bus=None, config=None):
         name = "ovos-PHAL-plugin-wallpaper-manager"
         super().__init__(bus=bus, name=name, config=config)
 
-        # this is a XDG compliant json storage object similar to self.settings in MycroftSkill
-        # it can be used to keep state
-        core_config = Configuration()
-        enclosure_config = core_config.get("gui") or {}
+        enclosure_config = self.config_core.get("gui") or {}
```

Comment:
while at it, can we drop this?

`self.active_extension` in the next line is not used anywhere, and doesnt even make sense anymore since it predates the GUIExtension becoming plugins